WordPress/src/main/res/layout/person_detail_fragment.xml

```diff
@@ -1,97 +1,63 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
     SlidingRelativeLayout is used for animating the slide in-out between fragments
-    Background color is set so the fragment below it, PeopleListFragment won't be visible
+    Background color is set so the fragment below it, PeopleListFragment, won't be visible
 -->
 <org.wordpress.android.ui.SlidingRelativeLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/default_background"
-    android:orientation="vertical">
+    android:background="@color/white"
+    android:clickable="true">
 
-    <RelativeLayout
+    <include
         android:id="@+id/person_details"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        layout="@layout/person_detail"/>
 
-        <org.wordpress.android.widgets.WPNetworkImageView
-            android:id="@+id/person_avatar"
-            android:layout_width="@dimen/avatar_sz_large"
-            android:layout_height="@dimen/avatar_sz_large"
-            android:layout_marginLeft="@dimen/margin_extra_large"
-            android:layout_marginTop="@dimen/margin_extra_large" />
+    <LinearLayout
```

Comment:
I don't think there is an issue here since it should be pretty obvious that you can't change the username of a user. I think we're also aiming for consistency with Calypso for the design, so when people come from Calypso, it'll immediately feel familiar. It's still worth pinging @rickybanister since he worked on the design.